core/src/test/scala/com/databricks/labs/remorph/discovery/SnowflakeTableDefinitionTest.scala

```diff
@@ -0,0 +1,27 @@
+package com.databricks.labs.remorph.discovery
+
+import com.databricks.labs.remorph.connections.{EnvGetter, SnowflakeConnectionFactory}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import java.sql.Connection
+class SnowflakeTableDefinitionTest extends AnyWordSpec with Matchers {
```

Comment:
also add unit test, where you use mocks for connections

Reply:
done